readme.md

```diff
@@ -56,10 +56,17 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
 - [7-Zip](http://www.7-zip.org/) (for deployment only)
 - [NSIS](http://nsis.sourceforge.net/) (for deployment only)
 
-### Mac:
+### macOS:
+
+- Xcode 8
+
+The program can also be built as a command line program using CMake. This type of build requires:
```

Comment:
I went ahead and added a message to the Xcode description mentioning the app bundle. I still think having the explanation that the CMake build create a "command line" binary is useful. Is there anything else I should add/change to make it clearer, though?